README.md

```diff
@@ -684,15 +684,21 @@ helper.getState(['query', 'attribute:*']);
 #### Get an object for the helper configuration in the query string
 
 ```js
-var state = algoliasearchHelper.AlgoliaSearchHelper.getConfigurationFromQueryString(qs);
+var state = algoliasearchHelper.url.getStateFromQueryString(qs);
 ```
 
 #### Get the configuration contained in the query string that is not for the helper
 
 ```js
-var otherConf = algoliasearchHelper.AlgoliaSearchHelper.getForeignConfigurationInQueryString(qs);
+var otherConf = algoliasearchHelper.url.getUnrecognizedParametersInQueryString(qs);
```

Comment:
Not bad. @redox did a little research yesterday on the topic and it seems that unrecognized should be pretty common.